.github/workflows/replace_links.yml

```
@@ -0,0 +1,34 @@
+name: Replace links with uids in copy branch
+
+on:
+  push:
+    branches: [ main ]
+    
+jobs:
+  build:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@master
+      with:
+        persist-credentials: false # otherwise, the token used is the GITHUB_TOKEN, instead of your personal token
+        fetch-depth: 0 # otherwise, you will failed to push refs to dest repo
+    - name: Commit changes
+      run: |
+        git config --local user.email "action@github.com"
+        git config --local user.name "GitHub Action"
+        $ThisCommit = "${{ github.sha }}"
+        git checkout ⭐Docs
+        git reset --hard $ThisCommit
+        pip install ./sync-script
+        replace_links https://raw.githubusercontent.com/geduardo/quantum-docs-pr/adding-file-script/articles/user-guide/language/uid-map.csv ./Specifications/Language/
```

This should absolutely work, but you probably shouldn't need to use `pip install` on `sync-script`; should be able to run in-place once you install `click` itself.

---

Do you mean this?
```suggestion
        pip install click
        python ./sync-script/replace_links.py https://raw.githubusercontent.com/geduardo/quantum-docs-pr/adding-file-script/articles/user-guide/language/uid-map.csv ./Specifications/Language/
```

And get rid of setup.py?